apps/guardian-ui/src/components/SetupProgress.tsx

```diff
@@ -111,6 +114,16 @@ export const SetupProgress: React.FC<SetupProgressProps> = ({
   isHost,
 }) => {
   const { t } = useTranslation();
+  const [isMobile, setIsMobile] = useState(window.innerWidth < 768);
+
+  useEffect(() => {
+    function handleResize() {
+      setIsMobile(window.innerWidth < 768);
+    }
+
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
```

Comment:
Use https://chakra-ui.com/docs/hooks/use-media-query or https://chakra-ui.com/docs/hooks/use-breakpoint-value instead, they've already got this figured out.

Reply:
But don't use either hook, see show / hide component below.

Reply:
updated